.gitignore

```diff
@@ -18,8 +18,9 @@ playground/dist
 .cache
 *.tsbuildinfo
 
-# Report generated from jest-junit
+# Report generated from tests
 test/native/junit.xml
+artifacts
```

Comment:
`artifacts` was added to this file in the meantime, you can delete this line or the other one.